apps/frontend/app/admin/contest/[id]/_components/Columns.tsx

```diff
@@ -68,11 +70,20 @@ export const columns = (
     accessorKey: 'score',
     header: () => <p className="text-center text-sm">Score</p>,
     cell: ({ row }) => (
-      <div className="flex justify-center">
+      <div
+        className="flex justify-center"
+        onClick={() => {
+          if (disableInput) {
+            toast.error('Problem scoring cannot be edited')
+          }
+        }}
+      >
         <Input
+          disabled={disableInput}
           defaultValue={row.getValue('score')}
           className="hide-spin-button w-[70px] focus-visible:ring-0"
           type="number"
+          style={{ pointerEvents: disableInput ? 'none' : 'auto' }}
```

Comment:
```suggestion
          disabled={disableInput}
          defaultValue={row.getValue('score')}
          className="hide-spin-button w-[70px] focus-visible:ring-0 disabled:pointer-events-none"
          type="number"
```

tailwind의 `disabled` 선택자를 사용하면 삼항연산자를 사용하지 않아도 될 것 같아요!

Reply:
수정했습니다!